TBODeveloperOverlayTests/NSString+RegexMatchesTests.m

```diff
@@ -1,76 +0,0 @@
-//
```

Comment:
But in the same file you added two new methods, that are not tested, as far as i can see...